### ISSUE TITLE
Decrease horizontal width

### ISSUE THREAD

**Issue by brettwooldridge** [hstove/issue_stats#23]
Pages with lots of badges are typically hard-pressed for horizontal real-estate, it would be great if the badge text were shortened.

##### From<br>
"Issues Closed In｜about 9 hours"

##### to →<br>
"issue closure｜9 hrs"

##### From<br>
"Pull Requests Closed In｜30 minutes"

##### to →<br>
"pull requests in｜30 min" ... or
"pull requests｜in 30 min" ... or
"pull closure｜30 min" ... or
"pulls｜in 30 min"

##### Basically:<br>
"about" → dropped
"less than a minute" → "< 1 min"
"minute(s)" → "min" (both singular and plural)
"hour(s)" → "hr/hrs"
"day(s)" → "d" (both singular and plural)
"month(s)" → "mon" (both singular and plural)
"year(s)" → "yr/yrs"

**Comment by hstove** [comment]
Since there are already multiple styles of badges available (flat and regular), this should be relatively easy to do. I don't have the time to do this unfortunately, but could point you in the right direction if you want to try contributing.

**Comment by brettwooldridge** [comment]
Sure, point away :+1:

**Comment by brettwooldridge** [comment]
I have submitted a pull request for your review.
